### PR TITLE
Release 2.1.0

### PR DESCRIPTION
**Pull request by georgehrke**
Requires:
- https://github.com/nextcloud/calendar/pull/2523

Would be nice to have:
- https://github.com/nextcloud/calendar/pull/2528
- https://github.com/nextcloud/calendar/pull/2476

### CODE REVIEW

**Comment by georgehrke** [comment]
@tcitworld Is there anything else you would like to get into 2.1.0? :)

**Comment by tcitworld** [comment]
Did a quick look in issues with 2.1 milestone and it seems fine apart from #2528 and #2476 indeed.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
+# 2.1.0 - 2020-09-02
+### Added
+- Dashboard integration
+  [#2414](https://github.com/nextcloud/calendar/pull/2414)
+- Better routes to link to calendar from outside
+  [#2483](https://github.com/nextcloud/calendar/pull/2483)
+- Different style for all-day / timed events
+  [#30](https://github.com/nextcloud/calendar/issues/30)
```
Comment (georgehrke) [inline CHANGELOG.md:8]: #2459

Comment (georgehrke) [reply CHANGELOG.md:8]: #8

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+
+### Fixed
+- Better localization of calendar-grid
+  [#1844](https://github.com/nextcloud/calendar/issues/1844)
+- Remove double scrollbars in Firefox
+  [#1815](https://github.com/nextcloud/calendar/issues/1815)
```
Comment (georgehrke) [inline CHANGELOG.md:14]: #2459

File: CHANGELOG.md (hunk 1 above)
Comment (georgehrke) [reply CHANGELOG.md:8]: #402

**Pull request merged**